examples/eval-tests.dx

```diff
@@ -153,6 +153,17 @@ cumsumplus : n=>Real -> n=>Real =
 :p [ 1 >= 2, 1.0 >= 2.0,  2 >= 2, 2.0 >= 2.0, 2 >= 1, 2.0 >= 1.0]
 > [False, False, True, True, True, True]
 
+infty = 1./0.
+:p infty
+> Infinity
+
+nan = 0./0.
+:p nan
+> NaN
```

Comment:
I don't know if it's a good idea to rely on this form of producing these, or this string representation in general. But I'm hoping it's adequate "for now".

Reply:
Definitely good enough for now. We could add dedicated primitives for them but I prefer to pretend they don't exist :stuck_out_tongue: